Makefile

```diff
@@ -1,6 +1,6 @@
 install: build-docs
-	rm -rf kedro/html
-	cp -r docs/build/html kedro
+	rm -rf kedro/framework/html
```

Comment:
I guess the other fix was to edit the CLI command code itself to look an extra parent above, but this works too - not sure what people prefer

Reply:
@mzjp2 Yes, you're right. I originally made that change, but then found out there are tests in the CLI command code which asserts that the `kedro/framework/html` is the correct location. Decided to do this change rather than change the test.